docs/reference/elasticsearch/rest-apis/index.md

```diff
@@ -16,3 +16,50 @@ This section includes:
 - [Common options](/reference/elasticsearch/rest-apis/common-options.md)
 - [Compatibility](/reference/elasticsearch/rest-apis/compatibility.md)
 - [Examples](/reference/elasticsearch/rest-apis/api-examples.md)
+
+## API endpoints
```

Comment:
We should get feedback from others but I think just a list of links might not be sufficient. I think we need to explanation for each category of API.

Reply:
For instance "You can use the following APIs to perform [autoscaling operations](https://www.elastic.co/guide/en/cloud/current/ec-autoscaling.html)."

Reply:
For data streams "For concepts and tutorials, see [Data streams](https://www.elastic.co/docs/manage-data/data-store/data-streams). " These are all additional benefits that are not shared in this link farm. This content should probably also live as an overlay in bump.sh at https://www.elastic.co/docs/manage-data/data-store/data-streams we just have a link that points back, we should have some actual blurb about what this category is.

Reply:
This I think is a much better example. https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-connector